.github/workflows/size.yml

```diff
@@ -56,8 +59,38 @@ jobs:
     - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
     - name: Run
       env:
-        OT_BASE_BRANCH: "${{ github.base_ref }}"
         GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
       run: |
         ./script/check-size
         cat /tmp/ot-size-report/report_pr >> $GITHUB_STEP_SUMMARY
+    - uses: actions/github-script@v7
+      name: Post Report
+      if: ${{ github.event_name == 'pull_request' }}
+      id: post-report
+      with:
+        script: |
+          const fs = require('fs')
+
+          const report = fs.readFileSync('/tmp/ot-size-report/report_pr', 'utf8');
```

Comment:
Let's use the generated report that follows the markdown table format, as it would look nicer. So let's use `report_table` instead of `report_pr`.

The `check-size` generates two different output files for different uses:
- `OT_REPORT_FILE_TABLE="${OT_TMP_DIR}/report_table"` has a nice markdown format to post as a comment on the PR.
- `OT_REPORT_FILE_PR="${OT_TMP_DIR}/report_pr"` is intended for output at the end of a PR run (in the job status), like this: [https://github.com/openthread/openthread/actions/runs/16165369712](https://github.com/openthread/openthread/actions/runs/16165369712)

Reply:
I triggered it to post two versions in https://github.com/bukepo/openthread/pull/19
it looks like the report_table is not accepted by GitHub. And the report_pr was used to post as a comment on the PR. Should we keep using it?